src/frontend/src/pages/WorkPackageDetailPage/WorkPackageViewContainer/WorkPackageViewContainer.tsx

```diff
@@ -166,8 +183,10 @@ const WorkPackageViewContainer: React.FC<WorkPackageViewContainerProps> = ({
       <WorkPackageDetails workPackage={workPackage} />
       <HorizontalList
         title={'Blocked By'}
-        items={workPackage.blockedBy.map((dep) => (
-          <strong>{wbsPipe(dep)}</strong>
+        items={dependenciesAsElements.map((dep) => (
+          <Link component={RouterLink} to={routes.PROJECTS + `/${wbsPipe(dep.wbsNum)}`}>
+            <strong>{`${wbsPipe(dep.wbsNum)} - ${dep.name}`}</strong>
```

Comment:
same comment as above fo rtypographys

Reply:
oops ig my other comment got deleted. I just said to replace the strong tag with a typography with a variant={‘bold’} because were trying to get away from using html tags